src/scripts/farming/Farming.ts

```diff
@@ -271,94 +271,94 @@ class Farming implements Feature {
             [
                 'This Berry has a refreshing flavor that strikes a good balance of dryness and sourness. It tastes better chilled.',
                 'This Berry slows the growth of nearby Berries.',
-            ], new Aura(AuraType.Growth, [0.9, 0.8, 0.7]), ['Snover', 'Flabébé (Blue)', 'Snom']);
+            ], new Aura(AuraType.Growth, [0.9, 0.8, 0.7]), ['Jynx', 'Swinub', 'Spheal', 'Snover', 'Flabébé (Blue)', 'Snom']);
```

Comment:
Gives another option to get QP pokemon if someone were willing to mutate that much of the farm in kanto instead of the small amount of QP